tdp/core/repository/git_repository.py

```diff
@@ -82,3 +82,9 @@ def is_file_modified(self, commit: str, path: Union[str, os.PathLike]) -> bool:
                 if diff.a_path == path or diff.b_path == path:
                     return True
             return False
+
+    def restore_file(self, file_names: str) -> None:
+        self._repo.index.checkout(paths=[file_names], force=True)
+
+    def reset(self) -> None:
```

Comment:
I do not see where `reset` is used.

Reply:
True, it was used at one point but I did not keep it.

I removed it from both `repository` and `git_repository`